app/src/main/java/me/ccrama/redditslide/SpoilerRobotoTextView.java

```diff
@@ -83,7 +59,9 @@
     private              List<Integer>        storedSpoilerStarts = new ArrayList<>();
     private              List<Integer>        storedSpoilerEnds   = new ArrayList<>();
     private static final Pattern              htmlSpoilerPattern  =
-            Pattern.compile("<a href=\"([#/](?:spoiler|sp|s))\">([^<]*)</a>");
+            Pattern.compile("<a.*href=\"[#/](?:spoiler|sp|s)\".*?>(.*?)</a>");
```

Comment:
It's important that the pattern does not track past the current tag, the `.*` after a will match greedily, for example the following will be caught in a single match: `<a href="#s">one</a> <a href="#s">two</a>`

A negative character class `[^>]*` would be the appropriate replacement since just making it lazy wouldn't handle `<a href="https://not-a-spoiler.com">one<a/> <a href="#s">two</a>` correctly